api/context.ts

```diff
@@ -23,14 +22,11 @@ export interface Context {
   ) => Promise<AuthenticateResponse>
   login: (user: User, options?: Record<string, unknown>) => Promise<void>
   logout: () => void
+  redis: Redis
 }
 
-export function buildContext(
-  { req, res }: ContextParams,
-  redis: Redis
-): Context {
+export function buildContext({ req, res, redis }: ContextParams): Context {
```

Comment:
I couldn't find any way to pass this redis client to buildcontext function to access it inside resolvers. So For the time being I hard-coded in the node-module. This function is from graphql-passport (node_modules/graphql-passport/lib/buildContext.d.ts). I think that is not the correct way to do this.

Reply:
I think it should work similar to the prisma integration:
1. Register it globally with tsryinge:
https://github.com/JabRef/JabRefOnline/blob/0d608bae290e416471c4cddb4471cf9a1b3cfafc/api/tsyringe.config.ts#L6-L8
2. Declare it as constructor dependency in services 
https://github.com/JabRef/JabRefOnline/blob/0d608bae290e416471c4cddb4471cf9a1b3cfafc/api/user/auth.service.ts#L16
3. Use it
https://github.com/JabRef/JabRefOnline/blob/0d608bae290e416471c4cddb4471cf9a1b3cfafc/api/user/auth.service.ts#L19

I think only the service (and not the resolvers) should know about redis since where the data is stored is somewhat of an implementation detail.